### DISCUSSION TITLE
v2.7 Not Showing via CDN link

### DISCUSSION THREAD

**Discussion by jonathanschimpf** [devicons/devicon#391] [Q&A] [answered]
Hey Guys! LOVE the recent update with new icon additions. I'm finding that the newly added v2.7 icons aren't displaying via the CDN link. I am using the font versions. Is there an updated link that may have not been added to the site? Or would I be better off installing via npm into my react-app for the newest icons? Thanks in advance!

**Comment by amacado** [comment, marked as answer]
Hello @jonathanschimpf, thanks for pointing out. I noticed that yesterday as well after publishing the new release. I think this is related due caching by jsdelivr.net. When I open the CDN link (https://cdn.jsdelivr.net/gh/devicons/devicon@master/devicon.min.css) now, it shows me the updated version 2.7. Can you confirm this? There is sadly now way in purging the cache by jsdelivr.net.

But it's on my list of todos to update the link in devicon.dev and not linking the `master` branch but instead linking to a specific release. This is better for backwards compatibility and should remove the caching issue. Since we're working on combining gh-pages branch and master branch this could be a enhancement. For now you can use the following CDN link (if above not working): https://cdn.jsdelivr.net/gh/devicons/devicon@v2.7/devicon.min.css


Btw: This is better placed in "issues" than in the "discussions".

**Reply by jonathanschimpf** [reply]
@amacado Thanks for your response. The CDN does show the latest updates when I view it, but doesn't work still when linking within my project. That additional link you provided worked immediately. I appreciate that. My apologies for the oversight when posting this Q&A here in discussions, it absolutely should exist in issues. Feel free to remove it if need be. Thanks again!
